### PR TITLE
Improve move contao section (hackaton, task 8)

### PR DESCRIPTION
**Pull request by m-vo**
This improves the "Move Contao" section.

* I wasn't really happy with the original document so I restructured it, made use of the tab feature and removed redundant content (see installation).
* I also adjusted the args of the dump/import commands because they were IMO easy to get wrong (no space in `-uUsername` for instance) or potentially generating bad files (missing `--hex-blob`).

These two things should get reviewed first IMO, because we likely want to port them to the German version as well.

Images will be adjusted by @stefansl in a followup PR.

### CODE REVIEW

File: docs/manual/installation/move-contao.en.md [hunk 1]
```diff
@@ -1,125 +1,102 @@
...
 {{% /notice %}}
 
-## Move Contao with the Contao Manager
+## Transferring the database
+### Export the database (source)
+You can either create a MySQL dump with the graphical database administration tool [phpMyAdmin](https://www.phpmyadmin.net/)
+or use the `mysqldump` program from the command line.
 
-### Export database on the local server
+{{< tabs >}}
```
Comment (m-vo) [inline docs/manual/installation/move-contao.en.md]: @fritzmg Would this need a tab group id? If so, how would this look?

(edit: I added one, as it seems necessary: https://learn.netlify.app/en/shortcodes/tabs/)

File: docs/manual/installation/move-contao.en.md [hunk 2]
```diff
@@ -1,125 +1,102 @@
...
+application related files.
 
-Moving a local installation to a live server is almost the same as a new installation.
+1. [Transferring the database](#transferring-the-database)
+2. [Transferring the files](#transferring-the-files)
+3. [Installing Contao](#installing-contao)
 
 {{% notice warning %}}
-To avoid inconvenience during the move, your local server should have the **[same PHP version](/en/installation/system-requirements/#minimum-php-requirements)** as running on the live server.
+To reduce the risk of conflicts, make sure your source and target server both run the **[same PHP version](/en/installation/system-requirements/#minimum-php-requirements)**.
```
Comment (netzarbeiter) [inline docs/manual/installation/move-contao.en.md]:
```suggestion
To reduce the risk of conflicts, make sure your source and target server both run the **[same PHP version](../system-requirements/#minimum-php-requirements)**.
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Moritz. Can you please adjust the german version too.

File: docs/manual/installation/move-contao.en.md (hunk 2 above)
Comment (m-vo) [reply docs/manual/installation/move-contao.en.md]: adjusted in 2570672

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**